addons/test_mail/tests/test_mail_message.py

```diff
@@ -148,6 +148,7 @@ def test_mail_message_values_fromto_long_name(self):
         done correctly (see ``_notify_get_reply_to_formatted_email`` docstring
         + commit linked to this test). """
         # name would make it blow up: keep only email
+        self.env.company.sudo().name = "Such long company name, much big"
```

Comment:
Normally it is accessible through "self.company_admin" so that it is consistent through mail tests (company_admin being the company of admin user whose company is used for employee; it is the same just following what we do in mail generally).

Reply:
Would move below " test_record.write({'name': 'Name that would be more than 78 with company name'})", as it is part of that test; the first one will anyway be more than 78 chars. And as we need 78 chars and the record name is about 49 (environ), we could name the company "Company name being about 33 chars"